server/meshmodel/aws-cloudfront-controller/relationships/hierarchical_inventory.json

```diff
@@ -0,0 +1,58 @@
+{
+  "apiVersion": "core.meshery.io/v1alpha1",
+  "kind": "Hierarchical",
+  "metadata": {
+    "name": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."
+  },
+  "model": {
+    "name": "aws-cloudfront-controller",
+    "version": "*",
+    "displayName": "AWS CloudFront",
+    "category": {
+      "name": "App Definition and Development",
+      "metadata": null
```

Comment:
Better to omit this line altogether, in this case.

@MUzairS15 @pandeyshubham03 what should we have in metadata here, in general?